test_scripts/Polices/Related_HMI_API/ATF_OnAllowSDLFunctionality_allowed_false_without_device.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------
+-- Description: 
+--     1. Preconditions: App is registered
+--     2. Steps: Activate App, send SDL.OnAllowSDLFunctionality with 'allowed=false' and without 'device' to HMI
+-- Requirement summary: 
+--     [Policy] "EXTERNAL_PROPRIETARY" flow: Related HMI API
```

Comment:
@vvvakulenko Please remove name of CRQ, this section is only for Requirements summary